tlsproxy.go

```diff
@@ -128,18 +139,20 @@ func doRun(listen func() (net.Listener, error), dial func() (net.Conn, error)) {
 	log.Debugf("Listening for incoming connections at: %v", l.Addr())
 
 	for {
-		in, err := l.Accept()
+		_in, err := l.Accept()
 		if err != nil {
 			log.Fatalf("Unable to accept: %v", err)
 		}
 
+		in := idletiming.Conn(_in, *idleTimeout, nil)
```

Comment:
Thinking again, it should be neglectable compared to the CPU required to handle TLS and copy data.